kore/src/Kore/Step/Simplification/CeilSimplifier.hs

```diff
@@ -105,3 +121,15 @@ runCeilSimplifierWith
 runCeilSimplifierWith ceilSimplifier env =
     runCeilSimplifier (hoistCeilSimplifier (flip runReaderT env) ceilSimplifier)
 {-# INLINE runCeilSimplifierWith #-}
+
+ceilSimplifierTermLike
+    ::  InternalVariable variable
+    =>  MonadSimplify simplifier
+    =>  CeilSimplifier
+            (ReaderT (SideCondition variable) simplifier)
+            (TermLike variable)
+            (OrCondition variable)
+ceilSimplifierTermLike =
```

Comment:
Where is this used, or where/why will this be needed?

Reply:
It's not, thanks for spotting that!